src/qt/createwalletdialog.h

```diff
@@ -23,6 +27,10 @@ class CreateWalletDialog : public QDialog
     explicit CreateWalletDialog(QWidget* parent);
     virtual ~CreateWalletDialog();
 
+#ifdef ENABLE_EXTERNAL_SIGNER
+    void setSigners(std::vector<ExternalSigner>& signers);
```

Comment:
24815c6309431cb0797defaf7add1150bcf4b567

nit `const &`.

Reply:
I'm adding that to: https://github.com/bitcoin/bitcoin/pull/21935